app/playground/usePlayground.ts

```diff
@@ -0,0 +1,44 @@
+import { useCallback } from 'react'
+import { I_ServiceApis } from '@/lib/homebrew'
+
+export const usePlayground = ({ services }: { services: I_ServiceApis | null }) => {
+  const fetchSettings = useCallback(async () => {
+    // Load the model from settings on page mount
+    const res = await services?.storage.getPlaygroundSettings()
+    const s = res?.data
+    return s
+  }, [services?.storage])
+
+  const loadModel = useCallback(async () => {
+    // Load the model from the bot settings on page mount.
+    const settings = await fetchSettings()
+    // Make payload
+    const selectedModelId = settings?.model?.id
+    const mode = settings?.attention?.mode
+    const initOptions = settings?.performance
+    const callOptions = {
+      model: 'local', // @TODO should load from a menu setting (global app setting ?)
+      ...settings?.response,
+    }
+    const listResponse = await services?.textInference.installed()
+    const installedList = listResponse?.data
+    const installPath = installedList?.find(i => i.id === selectedModelId)?.savePath
+    // Load LLM
+    const payload = {
+      modelPath: installPath,
+      modelName: settings?.model.name,
```

Comment:
Do we rly need to pass the name? Cant the server look this info up and record it when we make this call?